app/webpacker/src/javascripts/commodities.js

```diff
@@ -538,26 +538,36 @@
 
         $('.js-country-picker-select').each(function() {
           (function(element) {
+            let previousValue = element.find('option:selected').text();
             accessibleAutocomplete.enhanceSelectElement({
               selectElement: element[0],
               minLength: 2,
-              autoselect: true,
+              autoselect: false,
               showAllValues: true,
-              confirmOnBlur: true,
+              confirmOnBlur: false,
               alwaysDisplayArrow: true,
               displayMenu: 'overlay',
+              placeholder: previousValue, // in case the user clicks away without selecting anything
               dropdownArrow: function() {
                 return '<span class=\'autocomplete__arrow\'></span>';
               },
+              defaultValue: 'All countries',
               onConfirm: function(confirmed) {
-                const code = /\((\w\w)\)/.test(confirmed) ? /\((\w\w)\)/.exec(confirmed)[1] : null;
-                element.val(code);
-                const anchorInput = element.closest('.govuk-fieldset').find('input[name$="[anchor]"]');
-                anchorInput.val(window.location.hash.substring(1)); // maintain the tab
-                element.parents('form:first').trigger('submit');
+                const commodityCode = $('.commodity-header').data('comm-code');
```

Comment:
I know this feels frivolous but I'm wondering whether we can't extract all of this out into its own stimulus controller and write some unit tests?

I've tried it in the past and got waylaid but this file is just a bit horrible and is actually doing quite a lot of stuff